packages/ui/sass/components/badges.scss

```diff
@@ -0,0 +1,81 @@
+$badge-cdn-path: 'https://s3.amazonaws.com/cgcdn/global-ui/images/badges/';
+
+@mixin add-badge-background-image($name) {
+  @include add-background-image($badge-cdn-path, $name);
+}
+
+%badge-shared {
+  display: block;
+  outline: 0;
+  background-repeat: no-repeat;
+  background-position: 0;
+  background-size: contain;
+  width: 100px;
+  height: 100px;
+}
+
+.badge {
```

Comment:
@PRASHANTK22111993 I'd like the class names to more closely match what we have named the components in Figma. This will make it easier to find them in the future. Here's what is in figma:

- Badges
- Plates
- Badges with Plates

Badges in Figma are just the badge without a plate. Can you rename your classes to more closely match this convention? Maybe something like this:

`.badge` -> for badges without plates
`.badge-with-plate` - > badges with plates
`.plates` -> just the plates with no badge

Reply:
See figma for reference: https://www.figma.com/file/GH6DcTtUyYAnkaBQsfLOVgWm/CGUI-Global-Library?node-id=0%3A2656